services/communitytokens/service.go

```diff
@@ -68,14 +90,181 @@ func (s *Service) APIs() []ethRpc.API {
 
 // Start is run when a service is started.
 func (s *Service) Start() error {
+
+	s.walletWatcher = walletevent.NewWatcher(s.walletFeed, s.handleWalletEvent)
+	s.walletWatcher.Start()
+
 	return nil
 }
 
+func (s *Service) handleWalletEvent(event walletevent.Event) {
+	if event.Type == transactions.EventPendingTransactionStatusChanged {
```

Comment:
Since we only handle `EventPendingTransactionStatusChanged` here, you can reverse the condition and early exit for better readability.